source/_integrations/tuya.markdown

```diff
@@ -50,7 +50,7 @@ You have created an account for [Tuya Smart or Smart Life app](https://developer
 
 #### Create a project
 
-1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
+1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/?_source=627739af7d7e42379954f5c26a1c1759).
```

Comment:
We value privacy and should not add any form of tracking on links.

```suggestion
1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
```

Reply:
fixed.